docs/ldml/tr35-keyboards.md

```diff
@@ -1387,9 +1400,17 @@ Layers are matched exactly based on the modifier keys which are down. For exampl
 
 - `other` as a modifier will match if no other layers match.
```

Comment:
Just to clarify, do we support combining `other` with others? Like `shit others` to match both shift with anything else and shift alone.

Reply:
No, it would be `shift, other`